src/App.css

```diff
@@ -7,3 +7,10 @@
 .game-status-msg {
   margin-top: 0;
 }
+
+.game-change-settings-link{
+  font-size: 12px;
+  display: block;
+  padding: 10px 0 10px;
+  margin-bottom: -25px;
```

Comment:
Change the margin on the `container` instead

Reply:
Changing the margin for the `container` will also affect the `Landing` component, I wanted the settings link to be independent in it's styling. 
Additional margin should be added then to the `Landing` component, if it will be changed for the container, I'm not sure which option is better, your call...